contracts/PortfolioLib.sol

```diff
@@ -249,6 +250,9 @@ function getPoolReserves(PortfolioPool memory self)
     pure
     returns (uint128 reserveAsset, uint128 reserveQuote)
 {
+    // Check if -`self.liquidity` fits within an int128 by checking if it's
+    // greater than the minimum negative value of an int128.
```

Comment:
Strictly speaking, this comment is now inaccurate.

Reply:
I updated it by the way.